claim/netdata-claim.sh.in

```diff
@@ -286,6 +288,9 @@ if [ "${HTTP_STATUS_CODE}" = "204" ] ; then
         if [ "$EUID" == "0" ]; then
             chown -R "${NETDATA_USER}:${NETDATA_USER}" ${CLAIMING_DIR} || (echo >&2 "Claiming failed"; set -e; exit 2)
         fi
+        if [ "${RELOAD}" == "0"] ; then
+            exit 0
```

Comment:
Add `echo >&2 "Node was successfully claimed."` and see the shellcheck syntax error!

Reply:
whoops, no don't add the print :D

Reply:
I’ll check the spellcheck error tomorrow. Is there any point in outputting a success message when the script is called from the agent which is checking the return code?

Reply:
Hmm, you've got a point. I was thinking maybe somebody could call the script with this argument, but I guess why would he do that.

Reply:
Spell check is right about the missing space. I may feel compelled to fix it on the way to bed :)